lib/packwerk/run_context.rb

```diff
@@ -57,7 +57,7 @@ def initialize(
       node_processor_class: NodeProcessor,
       reference_lister: nil
     )
-      @root_path = File.expand_path(root_path)
+      @root_path = root_path
```

Comment:
It took me a little while to understand why this is okay, but I think it’s because a) the only place where `RunContext` gets constructed is in `RunContext.from_configuration`, which uses `Configuration#root_path`, and b) `Configuration` already calls `File.expand_path` in its constructor so `Configuration#root_path` has always been expanded.